portal/static/js/vendor/bootstrap_datatables_extension.js

```diff
@@ -42,15 +42,21 @@ var tnthTables = {
          * make sure the string passed in does not have line break element if so it is a possible mult-line text, split it up and use the first item in the resulting array
          */
         var regex = /<br\s*[\/]?>/gi;
-        a = a.replace(regex, "\n");
-        b = b.replace(regex, "\n");
-        var ar = a.split("\n");
-        if (ar.length > 0) {
-            a = ar[0];
+        //NOTE: adding if clause here because a is set to 0 if null/undefined, attempting to perform string specific operations,e.g. regular expression matching as follows, on 0, a numeric variable will result in runtime error
+        if (a) {
+            a = a.replace(regex, "\n");
+            var ar = a.split("\n");
+            if (ar.length > 0) {
+                a = ar[0];
+            }
         }
-        var br = b.split("\n");
-        if (br.length > 0) {
-            b = br[0];
+        //NOTE: adding if clause here because b is set to 0 if null/undefined, attempting to perform string specific operations,e.g. regular expression matching as follows, on 0, a numeric variable will result in runtime error
+        if (b) {
+            b = b.replace(regex, "\n");
+            var br = b.split("\n");
+            if (br.length > 0) {
+                b = br[0];
+            }
```

Comment:
I see use of both `a` and `b` below w/o similar null checks?

Reply:
thanks, paul, null checks added.